draft-ietf-rats-architecture.md

```diff
@@ -1328,16 +1328,21 @@ One way to provision key material is to first generate it external to the device
 In this case, confidentiality protection of the generator, as well as for the path over which the key is provisioned, is necessary.
 The manufacturer needs to take care to protect corresponding key material with measures appropriate for its value.
 
-Confidentiality protection can be realized via physical provisioning facility security involving no encryption at all. For low-security use cases, this might be simply locking doors and limiting personnel that can enter the facility. For high-security use cases, this might involve a special area of the facility accessible only to select security-trained personnel.
+The degree of protection afforded to this key material can vary by device, based upon considerations as to a cost/benefit evaluation of the intended function of the device.
+The confidentiality protection is fundamentally based upon some amount of physical protection: while encryption is often used to provide confidentiality when a key is conveyed across a factory, where the attestation key is created or applied, it must be available in an unencrypted form.
+The physical protection can therefore vary from situations where the key is unencrypted only within carefully controlled secure enclaves within silicon, to situations where an entire facility is considered secure,
+by the simple means of locked doors and limited access.
 
-Typically, cryptography is used to enable confidentiality protection. This can result in recursive problems, as the key material used to provision attestation keys must again somehow have been provisioned securely beforehand (requiring an additional level of protection, and so on).
+The cryptography that is used to enable confidentiality protection of the attestation key comes with its own requirements to be secured.
+This results in recursive problems, as the key material used to provision attestation keys must again somehow have been provisioned securely beforehand (requiring an additional level of protection, and so on).
 
-In general, a combination of some physical security measures and some cryptographic measures is used to establish confidentiality protection.
+So, this is why, in general, a combination of some physical security measures and some cryptographic measures is used to establish confidentiality protection.
 
-Another way to provision key material is to generate it on the device and export the validation key. If public-key cryptography is being used, then only integrity is necessary. Confidentiality of public keys is not necessary.
+When key material is generated within a device and the secret part of it never leaves the device,
+then the problem may lessen.  For public-key cryptography, it is, by definition, not necessary to
+maintain confidentiality of the public-key: however integrity of the chain of custody of the public is necessary in order to avoid attacks where an attacker is able to enroll an attestation key that they control.
```

Comment:
```suggestion
then the problem may lessen.  
For public-key cryptography, it is, by definition, not necessary to maintain confidentiality of the public-key: 
however integrity the public key is necessary in order to avoid attacks where an attacker is able to enroll an attestation key that they control.
```